modules/multi-architecture-scheduling-examples.adoc

```diff
@@ -0,0 +1,140 @@
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: kubernetes.io/arch
+                operator: In
+                values: <1>
+                - amd64
+                - arm64
+----
+<1> Specify the set of supported architecture. Valid values include `amd64` and `arm64`.
+
+Tainting every node for a specific architecture:: You can taint a node to avoid workloads that are not compatible with its architecture to be scheduled on that node. In the case that your cluster is using a `MachineSet` object, you can add parameters to a `.spec.template.spec.taints` field to avoid workloads being scheduled on nodes with non-supported architectures.
+
+* Before you can taint a node, you must scale down the `MachineSet` object or remove available machines. You can scale down the machine set with the following commands: 
++
+[source,terminal]
+----
+$ oc scale --replicas=0 machineset <machineset> -n openshift-machine-api
+----
++
+Or: 
++
+[source,terminal]
+----
+$ oc edit machineset <machineset> -n openshift-machine-api
+----
+For more information on scaling machine sets, see "Modifying a compute machine set".
+
++
+--
+.Example `MachineSet` with a taint set
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata: [...]
+spec:
+  [...]
+  template:
+    [...]
+    spec:
+      [...]
+      taints:
+      - effect: NoSchedule         
+        key: multi-arch.openshift.io/arch                
+        value: arm64
+----
+You can also set a taint on a specific node by running the following command: 
+[source,terminal]
+----
+$ oc adm taint nodes <node-name> multi-arch.openshift.io/arch=arm64:NoSchedule
+----
+--
+
+Creating a default toleration:: You can annotate a namespace so all of the workloads get the same default toleration by running the following command:
++
+[source,terminal]
+----
+$ oc annotate namespace my-namespace \
+  'scheduler.alpha.kubernetes.io/defaultTolerations'='[{"operator": "Exists", "effect": "NoSchedule", "key": "multi-arch.openshift.io/arch"}]'
+----
+
+Tolerating in workloads when the architecture taint is set:: On a node with a defined taint, workloads will not be scheduled on that node. However, you can allow them to be scheduled by setting a toleration in the pod specification.
++
+.Example deployment with a toleration 
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata: [...]
+spec:
+  [...]
+  template:
+    [...]
+    spec:
+      tolerations:
+      - key: "multi-arch.openshift.io/arch"
+        value: "arm64"
+        operator: "Equal"
+        effect: "NoSchedule"
+----
+
+This deployment can be used when you have an existing workload image that is not compatible with the architecture of a new node. This example deployment will also be allowed on nodes with the `multi-arch.openshift.io/arch=arm64` taint specified.
+//not sure if both worked or if there was just meant to be one description here 
```

Comment:
Hey I think the first statement (*This deployment can be used when you have an existing workload image that is not compatible with the architecture of a new node*)  is out of the scope

Reply:
Whoops! I meant to remove that in the previous update. Removing now!

Reply:
@aleskandro Didn't you mention something about that sentence here, https://github.com/openshift/openshift-docs/pull/63915#discussion_r1321830944.

Reply:
Hi @kelbrown20 @Prashanth684 The specific example we are discussing here adds toleration. I suggested the removal because, in this case, we allow the scheduling also on those nodes having the given taint, broadening the set of candidate nodes. Therefore, the user has (1) declared *to avoid* scheduling on the tainted nodes all the pods not tolerating for the taint, (2) added the toleration to a workload to declare that pod *supports/tolerate* the (tainted) architecture.

Reply:
ah..i see . By adding any sort of explanation we would probably be repeating what is already out there in the k8s docs, so makes sense to omit it.